sysbuild/Kconfig.netcore

```diff
@@ -28,6 +34,12 @@ config NETCORE_HCI_RPMSG
 	help
 	  Include hci_rpmsg as the netcore image to use
 
+config NETCORE_802154_RPMSG
+	bool "802154_rpmsg"
+	depends on SUPPORT_NETCORE_802154_RPMSG
+	help
+	  Include 802154_rpmsg as the netcore image to use
```

Comment:
If I understood your other comment correctly: https://github.com/nrfconnect/sdk-nrf/pull/11793#discussion_r1293337956

> if both bluetooth and 802.15.4 are required than this is preventing just showing the bluetooth image or just the 802.15.4 image, when it should only show the dual image option.

Shouldn't this also `depends on NRF_DEFAULT_802154`, so that the image is not displayed in menuconfig?

Reply:
No, `NRF_DEFAULT_802154` is for sample applications only